bin/svn-git-up

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Update Git and SVN repos checked out in the same directory.
+#
+# Pass a branch name like 4.6 as the sole argument to checkout that branch. Otherwise, master/trunk is checked out.
+
+cd $( git rev-parse --show-toplevel )
+
+if [[ ! -e .svn ]]; then
+	echo "Error: Unable to find svn root."
+	exit 1
+fi
+if [[ ! -e .git ]]; then
+	echo "Error: There is no git directory."
+	exit 1
+fi
+
+set -ex
+git fetch origin --tags
+
+git_branch=master
+svn_path='^/trunk'
```

Comment:
In a subsequent PR I want to improve this to use the currently branch as the default instead of `trunk`. This would allow you to switch to another branch via:

```sh
git svn-up 4.9
```

And then update that branch while it is checked out without also switching back to `trunk`:

```bash
git svn-up
```